Fix build failure caused by KVM-specific code in arches not supporting KVM:
  LINK  alpha-softmmu/qemu-system-alpha
i8259.o: In function `kvm_i8259_set_irq':
/tmp/portage/app-emulation/qemu-kvm-9999/work/qemu-kvm-9999/hw/i8259.c:689: undefined reference to `apic_set_irq_delivered'
diff --git a/hw/i8259.c b/hw/i8259.c
index 0632ea2..2f6789d 100644
--- a/hw/i8259.c
+++ b/hw/i8259.c
@@ -682,12 +683,14 @@ static int kvm_kernel_pic_load_from_user(PicState *s)
 
 static void kvm_i8259_set_irq(void *opaque, int irq, int level)
 {
+#ifdef CONFIG_KVM
     int pic_ret;
     if (kvm_set_irq(irq, level, &pic_ret)) {
         if (pic_ret != 0)
             apic_set_irq_delivered();
         return;
     }
+#endif
 }
 
 device_init(pic_register)
